include/SFML/Audio/SoundSource.hpp

```diff
@@ -55,6 +58,24 @@ class SFML_AUDIO_API SoundSource : AlResource
         Playing  //!< Sound is playing
     };
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Structure defining the properties of a directional cone
+    ///
+    /// Sounds will play at gain 1 when the listener
+    /// is positioned within the inner angle of the cone.
+    /// Sounds will play at outerGain when the listener is
+    /// positioned outside the outer angle of the cone.
+    /// The gain declines linearly from 1 to outerGain as the
+    /// listener moves from the inner angle to the outer angle.
+    ///
+    ////////////////////////////////////////////////////////////
+    struct Cone
+    {
+        Angle innerAngle;  //!< Inner angle, default is 360 degrees
+        Angle outerAngle;  //!< Outer angle, default is 360 degrees
+        float outerGain{}; //!< Outer gain, default is 1
```

Comment:
Same remark as Listener::Cone documentation: documented default values do not match default-constructed Cone values.